docs/api-reference.md

```diff
@@ -0,0 +1,138 @@
+# Rodux API Reference
+
+## Store
+The Store class is the core piece of Rodux. It is the state container that you create and use.
+
+### Store.new
+```lua
+Store.new(reducer, initialState, middlewares) -> Store
+```
+Creates and returns a new store.
+
+`reducer` is the store's root reducer function. `initialState` is the store's initial state. This should be used to load a saved state from storage. `middlewares` is a table of middlewares to apply.
+
+The store will automatically dispatch an initialization action with a `type` of `@@INIT`.
+
+!!! note
+	The initialization action does not pass through middlewares prior to reaching the reducer.
```

Comment:
It's kind of ambiguous >.<

"Why are the middleware in reverse order?" looks wrong (and "why is the middleware in reverse order" doesn't make it clear that the line refers to *all* the middleware, not just one).

Not sure how to resolve this.

Reply:
In cases where you would use "middlewares" you could probably use "any middleware" instead.

Man, the word "middleware" is kinda awful because of that, isn't it.